Interpolation, test cases, formatting, and decimal usage

ERJ21 opened the pull request:
Fixes: https://github.com/gojanpaolo/VDatumParser/issues/6

gojanpaolo commented:
Great work! :+1: 

Could you please also remove the "src/VDatumParser/bin"? I think it was added before we setup the .gitignore. Aside from that, I think this is good to go. :) Thank you!

ERJ21 commented:
Should be fixed now. Had an issue committing after using the remove command so I deleted the files manually and added that folder to the .gitignore just to be safe.
